pywt/tests/test_data.py

```diff
@@ -57,13 +58,22 @@ def test_wavelab_signals():
             assert_allclose(val, pywt.data.demo_signal(key),
                             rtol=rtol, atol=atol)
             assert_raises(ValueError, pywt.data.demo_signal, key, val.size)
-
         else:
             assert_allclose(val, pywt.data.demo_signal(key, val.size),
                             rtol=rtol, atol=atol)
             # these functions require a size to be provided
             assert_raises(ValueError, pywt.data.demo_signal, key)
 
+    # can get a list of the available signals
+    available_signals = pywt.data.demo_signal('list')
+    assert_('Doppler' in available_signals)
```

Comment:
How about in addition checking that `available_signals` matches `wavelab_result_dict.keys()`? That would prevent further issues with the npz file.